sys-cluster/mpich/mpich-3.2-r2.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+FORTRAN_NEEDED=fortran
+FORTRAN_STANDARD="77 90"
+
+inherit fortran-2 multilib-minimal mpi
+
+MY_PV=${PV/_/}
+DESCRIPTION="A high performance and portable MPI implementation"
+HOMEPAGE="http://www.mpich.org/"
+SRC_URI="http://www.mpich.org/static/downloads/${PV}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="mpich"
+KEYWORDS=""
+IUSE="+cxx doc fortran mpi-threads romio threads"
+REQUIRED_USE="mpi-threads? ( threads )"
```

Comment:
According to note at the end of the section at [1], it's preferable to just do the right thing when we know what the user wants.  In this case, I think the old ebuild got it right by enabling ```threads``` if ```mpi-threads``` was enabled.

1.  https://devmanual.gentoo.org/general-concepts/use-flags/index.html#conflicting-use-flags